streaming/vision/convert/README.md

```diff
@@ -0,0 +1,119 @@
+# Dataset preparation
+
+To use Streaming Dataset, We first convert the dataset from its native format to MosaicML's streaming dataset format (a collection of binary `.mds` files). Once in `.mds` format, we can store the dataset in a central location (filesystem, S3, GCS, or OCI) and stream the data to any compute cluster, with any number of devices, and any number of CPU workers, and it all just works.
+
+Follow the below steps to convert the Computer Vision dataset into a streaming MDS format. Also checkout the supported [MDSWriter()](https://streaming.docs.mosaicml.com/en/latest/api_reference/generated/streaming.MDSWriter.html) parameters for advanced usage.
```

Comment:
```suggestion
Check out steps below for information on converting common Computer Vision datasets to MDS format. Please see [MDSWriter()](https://streaming.docs.mosaicml.com/en/latest/api_reference/generated/streaming.MDSWriter.html) parameters for details on advanced usage.
```

Reply:
Addressed as part of https://github.com/mosaicml/streaming/pull/183/commits/4d242aaf5f26f73278a555a3021da6e328d0a28b.